storage-narpc/src/main/java/com/ibm/crail/storage/tcp/TcpStorageConstants.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Crail: A Multi-tiered Distributed Direct Access File System
+ *
+ * Author: Patrick Stuedi <stu@zurich.ibm.com>
+ *
+ * Copyright (C) 2016, IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.ibm.crail.storage.tcp;
+
+import org.slf4j.Logger;
+
+import com.ibm.crail.conf.CrailConfiguration;
+import com.ibm.crail.conf.CrailConstants;
+
+public class TcpStorageConstants {
+	public static final String STORAGE_TCP_INTERFACE_KEY = "crail.storage.tcp.interface";
```

Comment:
I still prefer to be able to specify an IP address the interface does not uniquely identify an IP